CODEOWNERS

```diff
@@ -1 +1,9 @@
-/src/lib/gadgets @o1-labs/crypto-eng-reviewers @mitschabaude
+# All files require a review from @o1-labs/o1js-admins
+* @o1-labs/o1js-admins
+
+# Changes to source code should be reviewed by exploration team and future zkapps team 
+/src/examples @o1-labs/exploration-team @o1-labs/future-zkapps
+/src/lib @o1-labs/exploration-team @o1-labs/future-zkapps
+
+# Provable code should be reviewed by the crypto team
+/src/lib/provable @o1-labs/crypto-eng-reviewers
```

Comment:
This is a legacy rule.  I'm happy to keep it, but I'm not sure if `crypto-eng-reviewers` is a valid target anymore.

Reply:
Let's remove this. `/provable` has a lot of stuff that won't require crypto engineer code reviews (eg the Merkle tree api and many other things)

Reply:
Is there any specific part of `/provable` that you would want a crypto review on?